CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [1.4.9]
+
+* AKS diagnostic bug resolved & storage diagnostic added.
+* Updated github auth scope to repo.
+* Adding flag to prevent draft from checking for manifests.
+* U/X add border for the row.
+* Fix the eslint no-unused-vars errors.
+* changing draft version to .38 and removing manual appname.
+* Dependabot updates and bumps.
+
```

Comment:
Tooltip changes for cluster properties page to display supported k8s version @Tatsinnit can you add this as well please?

Reply:
That tooltip changed was released as part of 1.4.8 here 

https://github.com/Azure/vscode-aks-tools/blob/main/CHANGELOG.md#148

please lete me know if I am missing something or add commit. Thanks

Reply:
Commit history confuses me , show tooltip changes were merged after 1.48 changelog PR , seems like 1.48 had some changes for tooltip and after that we had some enhancements to make it look like a table 1.49. 

![image](https://github.com/user-attachments/assets/1275de20-92f3-4ee4-a831-ab2ee2881bc6)

Reply:
Aha the one which adds ux border , ok done

Reply:
For example: 1.49

![image](https://github.com/user-attachments/assets/7212c042-214b-4ee8-9256-b6e69226a677)

1.48

![image](https://github.com/user-attachments/assets/f7caafa4-0bb5-4b80-8e7a-3baa7a86fffb)